portal-web/docroot/html/taglib/ui/page_iterator/start.jsp

```diff
@@ -112,6 +116,77 @@ NumberFormat numberFormat = NumberFormat.getNumberInstance(locale);
 
 				</ul>
 			</div>
+
+			<script data-senna-track="temporary" type="text/javascript">
+				var dropdown = document.getElementById("<%= ariaPagination %>");
+
+				var button = dropdown.querySelector('.dropdown-toggle');
+				var list = dropdown.querySelector('.dropdown-menu');
+
+				var options = list.querySelectorAll('.dropdown-item');
+				var selectedItemValue = button.dataset.attribute;
+
+				function onButtonKeyDown(event) {
+					if (event.key === 'ArrowDown' || event.key === 'ArrowUp' || event.key === 'Enter' || event.key === ' ') {
+						event.preventDefault();
+
+						button.setAttribute('aria-expanded', 'true');
+						list.classList.add('show');
+
+						var selectedOption = list.querySelector('.active');
+
+						if (selectedOption) {
+							selectedOption.focus();
+						}
+					}
+				}
+
+				button.addEventListener('keydown', onButtonKeyDown );
+
+				function onLeaveDropdown() {
+					button.setAttribute('aria-expanded', 'false');
+					list.classList.remove('show');
+				}
+
+				function handleKeyEvents(event) {
+					var currentIndex = Array.from(options).indexOf(document.activeElement);
+
+					if (event.key === 'ArrowDown') {
+						event.preventDefault();
+
+						if (currentIndex < options.length - 1) {
+							options[currentIndex + 1].focus();
+						}
+					} else if (event.key === 'ArrowUp') {
+						event.preventDefault();
+
+						if (currentIndex > 0) {
+							options[currentIndex - 1].focus();
+						}
+					} else if (event.key === 'Escape') {
+						button.focus()
+						onLeaveDropdown()
```

Comment:
Missing `;`s, `else` is not in new row. I would expect SF check to fail on this :thinking:

Reply:
That's weird, but all lack of commas and `else` on new lines were fixed